terraform/cf-certs/upload.tf

```diff
@@ -1,27 +0,0 @@
-provider "aws" {
-  region = "${var.region}"
-}
-
-resource "aws_iam_server_certificate" "system" {
-  name_prefix       = "${var.env}-system-domain-"
-  certificate_body  = "${var.system_domain_crt}"
-  private_key       = "${var.system_domain_key}"
-  certificate_chain = "${var.system_domain_intermediate_crt}"
-  path              = "/cloudfront/${var.env}-system-domain/"
-
-  lifecycle {
-    create_before_destroy = true
-  }
-}
```

Comment:
Can we leave this terraform in place in this PR. I think it should remain in place until we remove the certs that this manages.

Reply:
Did you check the PR #1211 where we add back the code in order to delete the certificate?

Reply:
TBH I do not think it should be in place, if we remove it we remove it. We can revert the code or use the code in the branch #1211 to remove this dependency. 

Otherwise, following the same reasoning, we should also keep the generation and upload of the self signed certificates, and if we do so, we had basically removed lots of work from the story and delayed to the follow up story.

Reply:
If we want to keep it, we should remove the commits that remove the generation & upload of apps&system domains: 43f221e404435a4bb36d8fb17ebc18467d6c5df4 and 53fcceb15f97c5357fe11b2bab82ff9bd5edacdd.

We can move them to #1211 or revert them and revert them back.

Reply:
Thinking about this further, I think it's fine as is...